carla/recourse_methods/catalog/clue/library/clue_ml/Clue_model/CLUE_counterfactuals.py

```diff
@@ -445,7 +446,7 @@ def update_stopvec(cost_vec, it_mask, step_idx, n_early_stop, min_steps):
         it_mask[to_mask == 1] = step_idx
 
         if (it_mask == 0).sum() == 0 and n_early_stop > 0:
-            print("it %d, all conditions met, stopping" % step_idx)
+            log.debug("iteration %d, all conditions met, stopping" % step_idx)
```

Comment:
Nitpick: Does the logger also outputs the module per default (e.g. CLUE_counterfactuals.py)? If not, then i think its not very-user intuitive which packages these logs actually belong to

Reply:
👍